src/test/java/org/jboss/qa/phaser/context/SimpleContext.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.qa.phaser.context;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SimpleContext implements Context {
```

Comment:
@osmman What about to put this simple implementation to phaser distribution? There is not implementation yet...